test/testTensor.cu

```diff
@@ -26,7 +26,7 @@ protected:
 
 TEMPLATE_WITH_TYPE_T
 void tensorConstructionZero() {
-    DTensor<T> zero(2, 3, 4, true);
+    DTensor <T> zero(2, 3, 4, true);
```

Comment:
why all the additional spaces in this file ?

Reply:
My stupid code formatter messed up